src/sqlfluff/dialects/dialect_oracle.py

```diff
@@ -16,10 +18,22 @@
 
 oracle_dialect.sets("unreserved_keywords").difference_update(["COMMENT"])
 oracle_dialect.sets("reserved_keywords").update(
-    ["COMMENT", "ON", "UPDATE", "INDEXTYPE"]
+    ["COMMENT", "ON", "UPDATE", "INDEXTYPE", "PROMPT"]
 )
 
 
+oracle_dialect.insert_lexer_matchers(
+        [
+            RegexLexer(
+                "prompt_command",
+                r"PROMPT([^(\r\n)]|)+((?=\n)|(?=\r\n))?",
```

Comment:
Rather than the initial `|`, could you not just change the `+` to `*`?

```suggestion
                r"PROMPT([^(\r\n)])*((?=\n)|(?=\r\n))?",
```

Reply:
+1 regex is greedy so the * should suffice